hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -364,7 +366,7 @@ private void setDataNodes(Map<String, String> dataNodes) {
    */
```

Comment:
not without having constant warnings from the compile.  This annotation signals that we are aware of it.